packages/plugins/gatsby-preview/admin/src/components/SyncSettings/index.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { Box } from '@strapi/design-system/Box';
+
+import { Grid, GridItem } from '@strapi/design-system/Grid';
+import { TextInput } from '@strapi/design-system/TextInput';
+import getTrad from '../../utils/getTrad';
+
+const SyncSettings = ({ contentSyncURL, formErrors, onChange }) => {
+  const { formatMessage } = useIntl();
+  const error = formErrors.contentSyncURL ? formatMessage(formErrors.contentSyncURL) : null;
+
+  return (
+    <Box paddingTop={7} paddingLeft={6} paddingRight={6} paddingBottom={11}>
+      <Grid>
+        <GridItem s={12} col={4}>
```

Comment:
I wonder if we could make it a bit wider?

```suggestion
        <GridItem s={12} col={8}>
```

because URLs are quite long and it's nice to see the full URL to review it.